pkg/service/update/service.go

```diff
@@ -0,0 +1,5 @@
+package updateBinary
+
+type updateBinary interface {
```

Comment:
where are you using this interface ..? and why do we need binary path..?

Reply:
Refactored